core/docker/bin/health-check

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function get_property() {
+    grep "^$1=" "$2" | cut -d'=' -f2
+}
+
+scheme=http
+port=8080
+
+config=/etc/trino/config.properties
+# prefer to use http even if https is enabled
+if [ "$(get_property 'http-server.http.enabled' "$config")" == "false" ]; then
+    scheme=https
+    port=$(get_property http-server.https.port "$config")
+else
+    port=$(get_property http-server.http.port "$config")
+fi
+
+endpoint="$scheme://localhost:$port/v1/info"
+
+# add --insecure to disable certificate verification in curl, in case a self-signed certificate is being used
+if ! info=$(curl --fail --silent --show-error --insecure "$endpoint"); then
+    echo >&2 "Server is not responding to requests"
+    exit 1
+fi
+
+if ! grep -q '"starting":false' <<<"$info" >/dev/null; then
```

Comment:
if JSON gets formatted with a space after colon, the healthcheck will always report healthy status immediately.
we may miss that. Would be safer to either use `jq`, or to require `"starting":true` (default to unhealthy when fmting changes)

Reply:
We don't have `jq` in the Docker image and I don't want to add it. How about adding `\s*` in the regexp?

Reply:
Done! I switched the condition and added `\s*`

Reply:
>  the healthcheck will always report healthy status immediately.
we may miss that

i was wrong here